index.html

```diff
@@ -3556,8 +3556,8 @@ <h5>JWT Encoding</h5>
             </ul>
 
             <p>
-For backward compatibility with JWT processors, the following JWT-registered
-claim names MUST be used instead of, or in addition to, their respective
```

Comment:
If we remove "or in addition to", it changes the normative meaning of this MUST statement in a way that will affect implementations. All of a sudden things that were legal, such as duplicating "iss" and "issuer" (which is mandatory in a VC), are not. We need to put "or in addition to" back if this is going to be an editorial change.

Reply:
This sentence is by far my least favorite normative statement I have ever read.

Reply:
But I agree with @msporny it needs to be fixed, and its not editorial.

Reply:
also, hidden in this change is date conversion between `issuanceDate` and `expirationDate` (XML Date Time) and `iat` and `exp` Unix Time Stamp.... thats enough to stop your heart.